connector/avro/src/main/scala/org/apache/spark/sql/avro/AvroOptions.scala

```diff
@@ -154,4 +157,5 @@ private[sql] object AvroOptions extends DataSourceOptions {
   // datasource similarly to the SQL config `spark.sql.avro.datetimeRebaseModeInRead`,
   // and can be set to the same values: `EXCEPTION`, `LEGACY` or `CORRECTED`.
   val DATETIME_REBASE_MODE = newOption("datetimeRebaseMode")
+  val STABLE_ID_FOR_UNION_TYPE = newOption("enableStableIdentifiersForUnionType")
```

Comment:
Can we add documentation for this? I think Spark conf version had long doc comment. We can reuse that here.